ArduPlane/tailsitter.cpp

```diff
@@ -39,10 +39,30 @@ bool QuadPlane::tailsitter_active(void)
  */
 void QuadPlane::tailsitter_output(void)
 {
-    if (tailsitter_active()) {
-        motors_output();
-        plane.pitchController.reset_I();
-        plane.rollController.reset_I();
+    if (!tailsitter_active()) {
+        return;
+    }
+
+    motors_output();
+    plane.pitchController.reset_I();
+    plane.rollController.reset_I();
+
+    if (tailsitter.input_mask_chan > 0 &&
+        tailsitter.input_mask > 0 &&
```

Comment:
using > 0 here excludes "-1". That's fine, just making sure that was intentional. The input_mask_chan is used as an index so that one should be enforced to >0 but input_mask is just checking for bits so it doesn't matter.